AmazonChimeSDK/AmazonChimeSDK/internal/ingestion/database/IngestionMeetingEvent.swift

```diff
@@ -13,5 +13,59 @@ import Foundation
 /// Swift doesn't decode Any type very well and it has to be one of pritimitve types.
 struct IngestionMeetingEvent: Codable {
     let name: String
-    let eventAttributes: IngestionEventAttributes
+    var eventAttributes: [String: AnyCodable?]
+}
+
+// MARK: Helper methods for retriving data from `eventAttributes`
+extension IngestionMeetingEvent {
+    
+    func getMeetingId() -> String? {
+        let item = eventAttributes[EventAttributeName.meetingId.description]
+        return item??.value as? String
```

Comment:
why two `??`?

Reply:
eventAttributes[EventAttributeName.meetingId.description] returns optional type, then  eventAttributes.value is also optional type, so need double unwrap.